bullet/src/SimulationFeatures.cc

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include "SimulationFeatures.hh"
+
+namespace ignition {
+namespace physics {
+namespace bullet {
+
+void SimulationFeatures::WorldForwardStep(
+    const Identity &_worldID,
+    ForwardStep::Output & /*_h*/,
+    ForwardStep::State & /*_x*/,
+    const ForwardStep::Input & _u)
+{
+    const WorldInfoPtr &worldInfo = this->worlds.at(_worldID);
```

Comment:
Should we check if the world pointer is valid before using it below?

Reply:
Mmm, AFAIK there's no way this function can get a wrong `_worldID` because the upper layers operate over registered `worldIDs`. Mind correcting me if I am wrong.

@azeey wdyt

Reply:
I remember a discussion between @scpeters  and @mxgrey that if the ID is not in the container, something has gone terribly wrong and so an exception would make sense. So, a lot of places in ign-physics follow the same pattern of calling `at`, even though it can throw.